server/routes/alertsRouter.ts

```diff
@@ -32,7 +35,7 @@ export default function alertsRouter(services: Services): Router {
     `${basePath}/alerts/active`,
     auditPageAccessAttempt({ services, page: Page.ActiveAlerts }),
     getPrisonerData(services),
-    permissionsGuard(services.permissionsService.getAlertsPermissions),
```

Comment:
The access code was just based on `getOverviewAccessStatusCode`

https://github.com/ministryofjustice/hmpps-prisoner-profile/pull/998/files#diff-436cd1a03284961f6db546c6d570d7f2d99d017aacdcf407b84fb925c79cfa47L38